bin/docker-healthcheck-coord

```diff
@@ -7,18 +7,18 @@ if [ ! -f "$bootstrapped" ]; then
     sleep_int=$(((RANDOM % sleep_int_max) + 1))
     echo "Sleeping to avoid race-condition (!) [$sleep_int s]"
     sleep "$sleep_int"
-    
```

Comment:
Not actually accidental, but fine. I'm slowly getting used to people stripping my whitespace out even when it makes sense conceptually. ;) I'll accept.

Reply:
Sometimes it really needs. For ex in Python in Pep8. But there I am not sure.

Reply:
Yes, but this is not Python. :) Shell scripts generally aren't whitespace-sensitive in that way (there are some exceptions, such as in tabs and multiline strings), but there's no harm in them if they make sense conceptually (e.g. everything is indented). But as I say, this doesn't really matter, here.